modules/cropReprojectLccAge/cropReprojectLccAge.R

```diff
@@ -139,7 +139,7 @@ cropReprojectLccCacheFunctions <- function(sim) {
 ### Inputs
 .inputObjects <- function(sim) {
   if (is.null(sim$age) | is.null(sim$lcc05)) {
-    checksums1 <- downloadData("LCC2005", file.path(modulePath(sim)))
+    checksums1 <- downloadData("LccToBeaconsReclassify", file.path(modulePath(sim)))
```

Comment:
I don't like that cropReprojectLccAge module is downloading the data from LccToBeaconsReclassify...these modules shouldn't depend on one another in this way

Reply:
Yes, you're right. I'll (try to) make it stand-alone